edward/criticisms.py

```diff
@@ -83,16 +83,16 @@ def evaluate(metrics, model, variational, data, y_true=None, n_samples=100):
     elif metric == 'squared_hinge':
       evaluations += [sess.run(squared_hinge(y_true, y_pred))]
     elif metric == 'mse' or metric == 'MSE' or \
-       metric == 'mean_squared_error':
+            metric == 'mean_squared_error':
```

Comment:
what's the pep8/tensorflow convention for these line breaks? is it 8 spaces, 2 spaces, 4 spaces, or should they be aligned with the `metric ==...` line above?